controller/lib/core/blower_fsm.h

```diff
@@ -155,11 +155,109 @@ class PressureControlFsm {
   Time expire_end_;
 };
 
+// "Breath finite state machine" for pressure assist mode.
+//
+// This mode is used for a patient that is not fully sedated and can initiate
+// breaths by himself, but once a breath has started, it is fully controlled
+// by the ventilator: inspiration time is set by the ventilator.
+// This mode enforce a minimum respiratory rate in case the patient does not
+// initiate enough breaths.
+// Similarly to pressure control, uses a square wave, only the length of the
+// exhale leg can vary depending on the patient breath initiation.
```

Comment:
This just means that the exhale leg of a previous breath will last until the patient (or the system) initiates the next one, right?